book/installation.rst

```diff
@@ -347,7 +347,7 @@ of the Symfony Installer anywhere in your system:
     c:\projects\> php symfony demo
 
 Once downloaded, enter into the ``symfony_demo/`` directory and run the PHP's
-built-in web server executing the ``php bin/console server:run`` command. Access
+built-in web server executing the ``php app/console server:run`` command. Access
 to the ``http://localhost:8000`` URL in your browser to start using the Symfony
 Demo application.
```

Comment:
I suggest to add a comment (http://docutils.sourceforge.net/docs/user/rst/quickref.html#comments) explaining why we use `app/console`, in order to avoid this regression in the future. What do you think @xabbuh ?